redbot/cogs/cleanup/cleanup.py

```diff
@@ -271,6 +271,40 @@ def check(m):
 
         await mass_purge(to_delete, channel)
 
+    @cleanup.command()
+    @commands.guild_only()
+    @commands.bot_has_permissions(manage_messages=True)
+    async def between(
+        self, ctx: commands.Context, one: int, two: int, delete_pinned: bool = False
```

Comment:
converters here should probably be changed to only match potentially valid snowflakes.

Reply:
Addressed in d19671f6a278e00eb143dcec7e4a666f6ca9bbd6.